examples/image-streams/image-streams-centos7.json

```diff
@@ -184,11 +212,15 @@
         "tags": [
           {
             "name": "latest"
+          },
+          {
+            "name": "9.2",
+            "from": {
+              "Kind": "ImageStreamTag",
+              "Name": "postgresql:latest"
```

Comment:
I was trying to set up the demonstration of having following tags, even if it is currently reversed... do you want me to just make it two independent tags?